### PR TITLE
init go modules for jaeger exporter

### LINKED ISSUE #205: Jaeger exporter should be its own go module
Among other things, this would decouple the core code from all the Jaeger dependencies, such as Thrift.

### PR DESCRIPTION
**Pull request by eran-levy**
fixes #205

### CODE REVIEW

File: exporter/trace/jaeger/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# OpenTelemtry-Go Jaeger Exporter
```
Comment (krnowak) [inline exporter/trace/jaeger/README.md]: Typo, should be OpenTelemetry-Go.

File: exporter/trace/jaeger/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+OpenTelemetry Jaeger exporter 
+
+## Installation
+```cassandraql
```
Comment (krnowak) [inline exporter/trace/jaeger/README.md]: cassandraql? I think you can just drop this word.

**Comment by eran-levy** [comment]
thanks @krnowak fixed, see last commits

**Comment by eran-levy** [comment]
@rghetia done signed the CLA, I see now "eran-levy authorized, but 1 other problem", cant understand from the details what is the other problem, do I miss something?

**Comment by rghetia** [comment]
> @rghetia done signed the CLA

Thanks. There is a conflict in go.mod. Can you please rebase/resolve the conflict?

**Comment by eran-levy** [comment]
> > @rghetia done signed the CLA
> 
> Thanks. There is a conflict in go.mod. Can you please rebase/resolve the conflict?

@rghetia done and checks passed

**Comment by krnowak** [comment]
Looks good.

**Pull request merged**